src/overlays/actors/ovl_Obj_Kibako/z_obj_kibako.c

```diff
@@ -8,7 +8,26 @@ void ObjKibako_Init(Actor* thisx, GlobalContext* globalCtx);
 void ObjKibako_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void ObjKibako_Update(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void ObjKibako_Draw(Actor* thisx, GlobalContext* globalCtx);
+void ObjKibako_SpawnCollectible(ObjKibako* this, GlobalContext* globalCtx);
+void func_809262BC(ObjKibako* this);
+void func_80926318(ObjKibako* this, GlobalContext* globalCtx);
+void ObjKibako_AirBreak(ObjKibako* this, GlobalContext* globalCtx);
+void ObjKibako_WaterBreak(ObjKibako* this, GlobalContext* globalCtx);
+void func_80926B40(ObjKibako* this);
+void func_80926B54(ObjKibako* this, GlobalContext* globalCtx);
+void ObjKibako_SetupIdle(ObjKibako* this);
+void ObjKibako_Idle(ObjKibako* this, GlobalContext* globalCtx);
+void ObjKibako_SetupHeld(ObjKibako* this);
+void ObjKibako_Held(ObjKibako* this, GlobalContext* globalCtx);
+void ObjKibako_SetupThrown(ObjKibako* this);
+void ObjKibako_Thrown(ObjKibako* this, GlobalContext* globalCtx);
+
+static UNK_TYPE2 D_80927380 = 0;
```

Comment:
Now that you have these decompiled, they should be using whichever type they actually are, so whatever UNK_TYPE2 expands to.